taskcluster/ci/ui-test/kind.yml

```
@@ -52,3 +52,42 @@ jobs:
                   path: /builds/worker/artifacts
                   type: directory
         worker-type: b-android
+    ui-test-screenshots-x86:
+        attributes:
+            build-type: debug
+        dependencies:
+            signing: signing-debug
+            signing-android-test: signing-android-test-debug
+        description: Run UI screenshots tests to keep them up to date
+        include-pull-request-number: true
+        #run-on-tasks-for: [github-pull-request, github-push]
```

Nit: Let's be explicit that we don't run this job in regular graphs.

```suggestion
        run-on-tasks-for: []
```